bedrock/firefox/templates/firefox/os/mwc-2015-preview.html

```diff
@@ -65,6 +65,27 @@
     </div>
   </section>
 
+  <section id="news">
+    <div class="container">
+      <div class="left">
+        <h3>{{ _('In the news') }}</h3>
+        <a href="">{{ _('See all press') }}</a>
```

Comment:
You mentioned this, but adding a note to ensure we remember to get the real link in here.